go/test/endtoend/vtgate/gen4/gen4_test.go

```diff
@@ -60,25 +60,6 @@ func TestCorrelatedExistsSubquery(t *testing.T) {
 	utils.AssertMatches(t, mcmp.VtConn, `select id from t1 where id in (select id from t2) order by id`,
 		`[[INT64(1)] [INT64(100)]]`)
 
-	utils.AssertMatches(t, mcmp.VtConn, `
-select id 
-from t1 
-where exists(
-	select t2.id, count(*) 
-	from t2 
-	where t1.col = t2.tcol2
-    having count(*) > 0
-)`,
-		`[[INT64(100)]]`)
-	utils.AssertMatches(t, mcmp.VtConn, `
-select id 
-from t1 
-where exists(
-	select t2.id, count(*) 
-	from t2 
-	where t1.col = t2.tcol1
-) order by id`,
-		`[[INT64(1)] [INT64(4)] [INT64(100)]]`)
```

Comment:
any reason for removing these test cases

Reply:
yeah, they are not valid. they are not following the `ONLY_FULL_GROUP_BY` directive, but this test doesn't change `sql_mode` on the connection, so the query fails.